Source/BSN.Commons.PresentationInfrastructure/Converters/JsonForceDefaultConverter.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace BSN.Commons.Converters
+{
+    public class JsonForceDefaultConverter<T> : JsonConverter<T>
```

Comment:
Please add comments for this class

Reply:
I resolved it

Reply:
What is meaning of default form?

Please describe them in comments

thanks :)

Reply:
The user must be know from your comments, that when he must use this class.

Reply:
I think ,I resolved it

Reply:
Please add example for this

Reply:
I hope ,this was resolved😢